### ISSUE TITLE
Train error on start

### ISSUE THREAD

**Issue by cka-14** [inspirehep/magpie#179]
Hi!

After running this code:

> from magpie import Magpie
> 
> magpie = Magpie()
> magpie.train_word2vec('data\\hep-categories', vec_dim=20)

I'm receiving this error: 

`<gensim.models.word2vec.Word2Vec at 0x1eeeedaba58>`

Is this a memory error or hard drive space error? Or completely something else? 

Thanks.

**Comment by jstypka** [comment]
that's an object that the method returns. You should assign it to a variable and then use it for Magpie training.

**Comment by cka-14** [comment]
Thanks for answer!

It was from readme so should i just assign variable to magpie? (Newbie in python)

And is there any long description/introduction about magpie?

**Comment by jstypka** [comment]
https://github.com/inspirehep/magpie/blob/master/README.md this should fix you up

**Comment by cka-14** [comment]
Thanks, was using this. I searched the issues to learn more about this topic. In readme file the commands are initiated from CLI with python, right? I'm using jupyter maybe that was a problem also.

Will try to do from CLI. On install I also got some errors like scipy wheel error (using windows 10) and installed manually all the dependencies with updated versions. Should I downgrade them? 
Thanks in advance.
